### PR TITLE
Replaces MATCHES -> STREQAL in the ctest cmake file

### PR DESCRIPTION
**Pull request by kyrsjo**
and add a forgotten test to the list of binary tests. This should make it pass on Windows.

### CODE REVIEW

**Comment by kyrsjo** [comment]
Done.

File: source/mod_hash.f90 [hunk 1]
```diff
@@ -141,20 +145,32 @@ subroutine hash_parseInputLine(inLine, iErr)
   select case(trim(lnSplit(1)))
 
   case("MD5SUM")
-    if(nSplit /= 2) then
-      write(lout,"(a,i3)") "HASH> ERROR MD5SUM expected 1 argument, got ",nSplit-1
+    if(.not. (nSplit.eq.2 .or. nSplit.eq.3)) then
```
Comment (vkbo) [inline source/mod_hash.f90]: That is a logical atrocity!
'if(nSplit /= 2 .and. nSplit /= 3) then'

Comment (vkbo) [inline source/mod_hash.f90]: That is a logical atrocity!
`if(nSplit /= 2 .and. nSplit /= 3) then`

**Comment by kyrsjo** [comment]
Thanks for the fixes and merging!

**Pull request merged**